OpenRA.Mods.Cnc/Traits/DrainPrerequisitePowerOnDamage.cs

```diff
@@ -0,0 +1,68 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
```

Comment:
2019.

Reply:
Also, this file didn't exist before 2019...